.github/workflows/ci-debian.yml

```
@@ -36,23 +36,31 @@ jobs:
       - name: Configure Debian
         id: conf
         run: cd ${{ env.WORK_DIR }};
-             ci/launch.sh conf;
+             ci/launch-debian.sh conf;
 
       - name: Launch system tests
         if: ${{ always() && steps.conf.conclusion == 'success' }}
         run: cd ${{ env.WORK_DIR }};
-             ci/launch.sh system;
+             ci/launch-debian.sh system;
 
       - name: Launch VM tests
         if: ${{ always() && steps.conf.conclusion == 'success' }}
         run: cd ${{ env.WORK_DIR }};
-             ci/launch.sh vm;
+             ci/launch-debian.sh vm;
 
-      - name: Upload test report
+      - name: Generate test report
         if: ${{ always() && steps.conf.conclusion == 'success' }}
         run: cd ${{ env.WORK_DIR }};
              git -C ci/test-report-pdf checkout 0156f2d35ba7d5983f22b6054aa83f92a42f0f2d;
-             ci/launch.sh report;
+             ci/launch-debian.sh report;
+
+      - name: Upload test report
+        if: ${{ always() && steps.conf.conclusion == 'success' }}
+        uses: actions/upload-artifact@v4
+        with:
+            name: seapath-test-report-yocto-standalone
```

Shouldn't be for Debian?
```suggestion
            name: seapath-test-report-debian-standalone
```

---

Yes indeed